src/components/FormLabel/FormLabel.scss

```diff
@@ -0,0 +1,16 @@
+@use '~@quartz/styles/scss/color-scheme';
+
+.container {
+	margin-bottom: 7px;
+
+	&.required {
+		display: flex;
```

Comment:
doesn't seem like we should need flexbox for this

Reply:
let me look

Reply:
hmm i think this is prob the most straightforward solution. we in effect get this markup:

```html
<div class="form-label">
  <div class="kicker">Label Text</div>
  <!-- ::after psuedo element lives here -->
</div>
```

in order to (1) keep FormLabel a block-level element and (2) not introduce another div, i think `display: flex` on the outer div is the best approach

Reply:
Gotcha, makes sense